tldr.py

```diff
@@ -17,7 +17,7 @@
 import colorama  # Required for Windows
 import shtab
 
-__version__ = "2.0.0"
+__version__ = "3.0.0"
```

Comment:
Is this right?

Reply:
Should be, if it's bumping to 3.x, any reason this wouldn't be right?